server/tests/integration/graphql.test.js

```diff
@@ -181,444 +181,444 @@ describe('graphql', () => {
-                        locationOfData
-                        testPlan {
-                            id
-                        }
-                        testPlanVersion {
-                            id
-                        }
-                        testPlanReport {
-                            id
-                        }
-                        at {
-                            id
-                        }
-                        browser {
-                            id
-                        }
-                        testPlanRun {
-                            id
-                        }
-                        test {
-                            id
-                        }
-                        scenario {
-                            id
-                        }
-                        assertion {
-                            id
-                        }
-                        testResult {
-                            id
-                        }
-                        atVersion {
-                            id
-                        }
-                        browserVersion {
-                            id
-                        }
-                        scenarioResult {
-                            id
-                        }
-                        assertionResult {
-                            id
-                        }
+                    id
+                    text
+                    atOperatingMode
+                  }
+                }
+                assertions {
+                  __typename
+                  id
+                  priority
+                  text
+                }
+                testFormatVersion
+              }
+            }
+            testPlanVersions {
+              id
+            }
+            issues {
+              __typename
+              author
+              title
+              link
+              isCandidateReview
+              feedbackType
+              isOpen
+              testNumberFilteredByAt
+              createdAt
+              closedAt
+              at {
+                name
+              }
+              browser {
+                name
+              }
+            }
+          }
+          testPlans {
+            directory
+            title
+          }
+          testPlanRuns {
+            id
+          }
+          testPlanVersions {
+            __typename
+            id
+            phase
+            draftPhaseReachedAt
+            candidatePhaseReachedAt
+            recommendedPhaseTargetDate
+            recommendedPhaseReachedAt
+            deprecatedAt
+            versionString
+          }
+          testPlanVersion(id: 1) {
+            __typename
+            id
+            testPlanReports {
+              id
+            }
+            testPlan {
+              id
+              directory
+            }
+          }
+          recommendedTestPlanVersion: testPlanVersion(id: 67) {
```

Comment:
For reviewers: Formatting has messed with this diff. This id was changed to 67 from 69